migrating_from_ocp_3_to_4/planning-migration-3-4.adoc

```diff
@@ -176,21 +176,21 @@ Review the following logging changes to consider when transitioning from {produc
 
 {product-title} 4 provides a simple deployment mechanism for OpenShift Logging, by using a Cluster Logging custom resource.
 
-For more information, see xref:../logging/cluster-logging-deploying.adoc#cluster-logging-deploying_cluster-logging-deploying[Installing OpenShift Logging].
+For more information, see xref:../observability/logging/cluster-logging-deploying.adoc#cluster-logging-deploying_cluster-logging-deploying[Installing OpenShift Logging].
 
 [discrete]
 ==== Aggregated logging data
 
 You cannot transition your aggregate logging data from {product-title} 3.11 into your new {product-title} 4 cluster.
 
-For more information, see xref:../logging/cluster-logging.adoc#cluster-logging-about_cluster-logging[About OpenShift Logging].
+For more information, see xref:../observability/logging/cluster-logging.adoc#cluster-logging-about_cluster-logging[About OpenShift Logging].
```

Comment:
Thanks for your [comment](https://github.com/openshift/openshift-docs/pull/74819#issuecomment-2072433773).

> Some friendly reference text links are not 100% accurate. These might not need changed, but I am just noting it. For example, one might say "See About Logging," but it might actually link to a section called "OpenShift Container Platform Logging". This just an example. I noted various instances of this. Maybe they don't need changed since you didn't change the friend reference text.

So, I am not making updates here.